### DISCUSSION TITLE
Add Chinese Store Description

### DISCUSSION THREAD

**Discussion by yinyue200** [File-New-Project/EarTrumpet#654] [Ideas]
Store Description for Simplified Chinese:
```
EarTrumpet 是一个用于传统和 Windows 应用商店应用的音量控制应用。
```
Store Description for Traditional Chinese:
```
EarTrumpet 是一個用於傳統和 Windows 應用商店應用的音量控制應用。
```

Thanks

**Comment by riverar** [comment]
@yinyue200 Our text has changed, if you could update the description, that'd be great!

**Comment by yinyue200** [comment]
Simplified Chinese: EarTrumpet 是一款功能强大的 Windows 音量控制软件

Traditional Chinese: EarTrumpet 是一款功能強大的 Windows 音量控制應用程序

**Comment by Geeyun-JY3** [comment]
Any progress on this issue(discussion)? Since @yinyue200 [has updated the text](https://github.com/File-New-Project/EarTrumpet/discussions/654#discussioncomment-333469) to match EarTrumpet's new Store Description.

**Reply by riverar** [reply]
We have a few more strings that need translation before we can send it to the Store. I'll add them to CrowdIn now, standby.

**Reply by Geeyun-JY3** [reply]
I have translated it in Crowdin (but only Simplified Chinese🤣). Thanks @yinyue200 for contributing!

**Reply by yinyue200** [reply]
Now it have translation for Traditional Chinese too.
